workflow/rules/ref.smk

```diff
@@ -101,3 +101,14 @@ rule update_lineages:
         "logs/{date}/pangolin/update.log",
     shell:
         "(mkdir -p {output} && wget -qO- https://github.com/cov-lineages/lineages/archive/master.tar.gz | tar xvz --strip-components=1 -C {output})> {log} 2>&1"
+
+
+rule get_gisaid_provision:
+    output:
+        temp("resources/gisaid/provision.json"),
+    params:
+        user_and_pw=get_gisaid_auth(),
```

Comment:
This is not a good idea, because the PW will be persisted in Snakemake's metadata and seen in the logs etc. Better use environment variables for this: https://snakemake.readthedocs.io/en/stable/snakefiles/configuration.html?highlight=envvars#environment-variables